content/docs/learning-guide/tutorials/assets/mesh-assets.md

```diff
@@ -2,69 +2,64 @@
 
-1. The customizations you make in Scene Settings are stored in a *sidecar file* with a `.assetinfo` extension. When Asset Processor detects a `.assetinfo` file, it uses the settings in the file to process the related source asset. This sidecar file is treated as a source dependency for the asset. This means that if the `.assetinfo` file is changed, the source asset will be reprocessed even if the source asset has not changed.
+    The **Select meshes** property reads **All meshes selected**. You can choose the {{< icon browse-edit-select-files.svg >}} **node select** button to select which meshes to include in the mesh group.
 
-    Let's add a modifier to customize how the asset is processed. Choose the **Add Modifier** button to view the mesh modifier list and select **Coordinate system change**.
+1. Add a modifier to customize how the asset is processed. Choose the **Add Modifier** button to view the mesh modifier list and select [**Coordinate system change**](/docs/user-guide/assets/scene-settings/meshes-tab#coordinate-system-change).
 
     ![ Scene Settings meshes tab, adding mesh a modifier. ](/images/learning-guide/tutorials/assets/meshes-coordinate-system-change.png)
 
 1. The Coordinate system change mesh modifier is used to scale or transform the asset for those scenarios where the asset might be too small, too large, or incorrectly oriented in O3DE. By default, the modifier provides a single option to rotate the mesh 180 degrees. Activate the **Use advanced settings** toggle to expose the advanced modifier settings.
 
     ![ Scene Settings meshes Coordinate system change modifier, Use advanced settings. ](/images/learning-guide/tutorials/assets/meshes-use-advanced-settings.png)
 
-1. Let's customize the scale of the asset. Set the **Scale** property to `5.0` to scale the asset to five times its size.
-
-    {{< note >}}
-You can learn about the advanced settings properties of the Coordinate system change modifier, as well as the other available modifiers and their properties, in the [Scene Settings](/docs/user-guide/assets/scene-settings) topic of the **User Guide**.
-    {{< /note >}}
+1. Customize the scale of the asset. Set the **Scale** property to `5.0` to scale the asset to five times its size.
 
 1. Choose the **Update** button at the bottom-right of Scene Settings. This creates or updates the `.assetinfo` sidecar file and triggers Asset Processor to reprocess the asset.
 
+    When Asset Processor detects a `.assetinfo` file, it uses the settings in the file to process the related source asset. This sidecar file is treated as a source dependency for the asset. This means that if the `.assetinfo` file is changed, the source asset will be reprocessed even if the source asset has not changed.

```

Comment:
question: It does seem there is some duplication between mesh and actor asset docs, I wonder if (maybe in future) it could be combined to make it harder for these two things to get out of sync, but it's not a massive problem.

Reply:
all the .assetInfo file behave the same. Maybe we could have a common practice page to talk about this.